### ISSUE TITLE
KDE install fails again on selecting non existing kde-l10n

### ISSUE THREAD

**Issue by killajoe** [Antergos/Cnchi#997]
KDE install fails again on selecting non existing kde-l10n

-Booting latest Antergos ISO,
-choose to install KDE and choose AUR support as extra option (advanced option) 
-choose englisch-GB as language
(reproduced test on virtualbox/EFI)

install fails on missing package **kde-l10n-en_gb**

Full log: (/tmp/cnchi.log)
https://ptpb.pw/1QjX

Report started on Antergos Forum Post:

https://forum.antergos.com/topic/10573/installing-kde

**Comment by karasu** [comment]
Latest version is 0.14.475

From some reason, Cnchi didn't update itself...

**Comment by karasu** [comment]
Yep, update it's not working and even worse, cnchi claims it's at it latest version.
